### PR TITLE
add: tools.md

### LINKED ISSUE #161: Create a "support section" validator
Create a simple NPM module that will confirm that the value of the `support` property in the `package.json` file matches one of the established values from the "support level" documentation created in this PR -> #139

### PR DESCRIPTION
**Pull request by Eomm**
I would add this list in order to track the future tools that will be built.

If you think that is better to transfer the module from my profile to the Node.js, it is super ok 👍

Closes #161

### CODE REVIEW

File: docs/drafts/tools.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Tools
+
+The tools builded by the package maintenance group to ease the adoption of the best practice defined
+by this working group:
+
+- [`package-complian`](https://github.com/Eomm/package-compliant): validate the `support` section
```
Comment (ljharb) [inline docs/drafts/tools.md]:
```suggestion
- [`package-compliant`](https://github.com/Eomm/package-compliant): validate the `support` section
```

File: docs/drafts/tools.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Tools
+
+The tools builded by the package maintenance group to ease the adoption of the best practice defined
```
Comment (rxmarbles) [inline docs/drafts/tools.md]: `The tools builded by` => `The tools built by`  & `of the best practice` => `of the best practices`

File: docs/drafts/tools.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+by this working group:
+
+- [`package-compliant`](https://github.com/Eomm/package-compliant): validate the `support` section
```
Comment (rxmarbles) [inline docs/drafts/tools.md]: `validate` => `validates`

Comment (Eomm) [reply docs/drafts/tools.md]: Done 👍

**Pull request merged**